### PR TITLE
[4.0] Creating RTL CSS for Joomla installation

### PR DESCRIPTION
**Pull request by infograf768**
As title says

### Summary of Changes
Added rtl scss file
Normalised name of rtl.css
Load the rtl css when language is rtl
npm install will now create the rtl files including a minified one for Joomla Installation

### Testing Instructions
Change en-GB.xml to `<rtl>1</rtl>`
Install a clean joomla.

### CODE REVIEW

File: installation/template/scss/template-rtl.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
+@import "template";
+
+body {
+  direction: rtl;
+}
+// Javascript Warning
+#javascript-warning {
```
[the remark this answers is not in the document]
Comment (infograf768) [reply installation/template/scss/template-rtl.scss:7]: That selector was present in the former template_css.rtl (which was not used)

File: installation/template/scss/template-rtl.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+.j-header-help {
+  margin: 20px auto 20px 20px;
+}
+
+.j-footer {
+  text-align: left;
+}
+
+.fa-chevron-right:before {
```
[the remark this answers is not in the document]
Comment (infograf768) [reply installation/template/scss/template-rtl.scss]: No idea what you mean here, my dear hound... I can see that nowhere in core including for fontawsome

**Comment by infograf768** [comment]
@wilsonge @laoneo

File: installation/template/scss/template-rtl.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (brianteeman) [reply installation/template/scss/template-rtl.scss]: Change to 
`.fa-chevron-right::before {`

[the remark this answers is not in the document]
Comment (infograf768) [reply installation/template/scss/template-rtl.scss]: maybe, dear hound, you should read
https://stackoverflow.com/questions/10181729/should-i-use-single-or-double-colon-notation-for-pseudo-elements
😸

[the remark this answers is not in the document]
Comment (brianteeman) [reply installation/template/scss/template-rtl.scss]: Never rely on stackoverflow for an answer.

https://www.w3.org/TR/selectors/#pseudo-element-syntax

[the remark this answers is not in the document]
Comment (infograf768) [reply installation/template/scss/template-rtl.scss]: it is much more secure to use single colon as that is what we also use in core.

[the remark this answers is not in the document]
Comment (brianteeman) [reply installation/template/scss/template-rtl.scss]: There are 
712 instances of ::before
504 instances of ::after

**Comment by laoneo** [comment]
I do always write a single :. But reading now the link from @brianteeman double is the way to go as single is deprecated.

**Comment by infograf768** [comment]
I changed MY unique stuff.
I see that in the installation template.css there are no occurence of double ::

File: installation/template/scss/template-rtl.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (infograf768) [reply installation/template/scss/template-rtl.scss:7]: can we keep it please until we have a specific class in the tmpl?
@laoneo 
I just wanted to do the rtl job here.

**Pull request merged**